agent/structs/acl.go

```diff
@@ -1077,8 +1077,9 @@ func (r *ACLTokenBatchGetRequest) RequestDatacenter() string {
 // This is particularly useful during token replication and during
 // automatic legacy token upgrades.
 type ACLTokenBatchSetRequest struct {
-	Tokens ACLTokens
-	CAS    bool
+	Tokens            ACLTokens
+	CAS               bool
+	AllowMissingLinks bool
```

Comment:
During server version skew there is a window of time when an upgraded server will be interpreting structs from an older server that was assuming `true` but the new server will read the absence of the value as `false`.

If we think this is going to be a problem I could invert the logic here and have `ProhibitMissingLinks` instead.

Reply:
I guess this also applies when reading old RAFT entries, so probably I'll have to do that.

Reply:
I think how you have it is fine. It should have errored before but now will work as expected. That and its unlikely to hit the race condition between the RPC endpoint and raftApply anyways.